sfyra/cmd/sfyra/cmd/bootstrap_capi.go

```diff
@@ -33,9 +33,9 @@ var bootstrapCAPICmd = &cobra.Command{
 
 				RegistryMirrors: options.RegistryMirrors,
 
-				CPUs:   options.BootstrapCPUs,
-				MemMB:  options.BootstrapMemMB,
-				DiskGB: options.BootstrapDiskGB,
+				BootstrapCPUs:   options.BootstrapCPUs,
```

Comment:
change in `sfyra`: migrate from creating two clusters - one for Sidero Metal (management), another for PXE VMs down to a single cluster: two VMs run Talos + Sidero Metal, others are empty PXE VMs.